apps/site/assets/css/_events.scss

```diff
@@ -324,10 +320,21 @@ $mobile-control-height: 73px;
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
+
+      &:hover, &:focus {
+        color: $brand-primary;
+        text-decoration: none;
+      }
     }
   }
 }
 
+button[data-js='enabled'] {
+  body.no-js & {
+    display: none;
+  }
+}
+
```

Comment:
This can be more simply written as

```scss
.no-js button[data-js='enabled'] {
  display: none;
}
```

I also think it should be a bit more specific, maybe to the calendar page:

```scss
.no-js .m-event-calendar button[data-js='enabled'] {
  display: none;
}
```